projects/application/HashMix-Lending.md

```diff
@@ -0,0 +1,71 @@
+## Project Name
+
+HashMix Lending
+
+## Category 
+
+Application
+
+## Project Description
+
+HashMix proposes a tokenization scheme suitable for arbitrary hash power, implements a set of universal decentralized cloud hash power trading, lending, and swapping protocols to solve hash power fraud, hash power oversold, over-simplified product design, and lack of liquidity and flexibility in the current market, enables the free redistribution of global hash power.
+
+The HashMix project got support from some of the top investment institutes, including HashKey Capital, Fenbushi Capital, LongHash Ventures, GBV, Kenetic Capital, and so on.
+
+The first product HashMix launched is HashMix Lending, a lending tool helping Filecoin miners and FIL holders. Miners can pledge their miner account to borrow FIL to seal more sectors, while FIL holders can deposit FIL to receive an awesome interest. 
+
+## Project Status
+
+HashMix Lending has been launched in July, 2021, currently 376k FIL have been deposited on HashMix platform.
+
+## Previews
+
+![image](https://user-images.githubusercontent.com/86829888/128304495-dc30e847-5766-401b-bffc-7c52a568c2fa.png)
+
+## Target Audience
+
+Filecoin miners who need FIL to seal more storage power and FIL holders who want to fully utilize their assets.
+
+## Rough estimated user base (if applicable)
+
+Every Filecon miner and holder could benefit from HashMix platform.
+
+## Github repo
+
+https://github.com/HashMixProject
+
+
+## Website
+
+https://hashmix.org/
+https://app.hashmix.org/
+
+## Docs
+
+Coming soon.
+
+
+## Team Info
+
+The HashMix team is anonymous.
+
+
+## Team Size  
+
+20
+
+## How the community can engage
+
+GitHub Discussion: https://github.com/filecoin-project/community/discussions/223  
+Email: contact@hashmix.org  
+Twitter: https://twitter.com/HashMixOfficial
```

Comment:
```suggestion
Twitter: https://twitter.com/HashMixOfficial  
Discord: https://discord.gg/VDHDaSR7D9  
Telegram: https://t.me/HashMixChannel  
Medium: https://hashmix.medium.com/  
```

Reply:
It will break each option to a new line.

Reply:
Fixed.

Reply:
> It will break each one into a new line.

Fixed.